games/devtest/mods/unittests/async_env.lua

```diff
@@ -207,3 +207,37 @@ local function test_vector_preserve(cb)
 	end, {vec})
 end
 unittests.register("test_async_vector", test_vector_preserve, {async=true})
+
+local function fill_async()
```

Comment:
personally I'd write this down
```suggestion
-- FIXME: this code is racy and can be improved once Lua IPC is supported
local function fill_async()
```